force-app/main/default/aura/resourceStatusTracker/resourceStatusTrackerHelper.js

```diff
@@ -26,7 +26,8 @@
 
                 if(data != null){
                     component.set("v.statusReport", data);
-                    component.set("v.count", data.Count__c);
+                    let count = component.get("v.statusReport.Count__c");
```

Comment:
is it possible that this needs namespace prefix? component.get("v.statusReport.tractionthrive__c");

Reply:
lets try this fix